org.munadi.Munadi.appdata.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Submit this file upstream please.

Reply:
Will do thanks. And yes, I am the author of the app.